include/aca_zeta_programming.h

```diff
@@ -27,11 +27,28 @@ static atomic_uint current_available_group_id(1);
 
 namespace aca_zeta_programming
 {
+struct fwd_info {
+  string ip_addr;
+  string mac_addr;
+  fwd_info(string ip_addr = "", string mac_addr = "")
+          : ip_addr(ip_addr), mac_addr(mac_addr){};
+
+  // Overload "=="for hash operation
+  bool operator==(const fwd_info &other) const
+  {
+    if (ip_addr == other.ip_addr && mac_addr == other.mac_addr) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+};
 struct zeta_config {
   uint group_id;
   uint oam_port;
-  // set<gateway_node_ip_address>
-  unordered_set<string> zeta_buckets;
+
+  // CTSL::HashMap <key: fwd_info, value: int* (not used)>
+  CTSL::HashMap<fwd_info *, int *> zeta_buckets;
```

Comment:
is this hashing based on the pointer address of "fwd_info *"? correct me if I am wrong.